### PR TITLE
use mdapi instead of repoquery

### PR DESCRIPTION
**Pull request by ignatenkobrain**
Closes: #127
Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>

### CODE REVIEW

File: hotness/consumers.py [hunk 1]
```diff
@@ -221,21 +219,19 @@ def _handle_anitya_update(self, upstream, package, msg):
...
-            # likely hasn't yet been a rawhide build of it.. and there's
-            # nothing reasonable we can do.  Notify the world of our failure
-            # and go back to the event loop.
-            self.log.warn("No rawhide version found for %r" % package)
+        mdapi_url = '{0}/srcpkg/koji/{1}'.format(self.mdapi_url, package)
+        self.log.debug("Getting pkg info from %r" % mdapi_url)
+        r = requests.get(mdapi_url)
+        if r.status != 200:
+            # Unfortunately it's not in mdapi, we can't do much about it
+            self.log.warning("No koji version found for %r" % package)
```
Comment (pypingou) [inline hotness/consumers.py]: Should we fall back to the rawhide repo then?

Comment (ignatenkobrain) [inline hotness/consumers.py]: I don't think so. If package not in koji repo, then I don't see point why we should touch rawhide. I guess this is the case for retired packages.

Comment (pypingou) [inline hotness/consumers.py]: Does the koji repo contains all the packages?

Comment (ignatenkobrain) [inline hotness/consumers.py]: I think it does.

**Comment by bowlofeggs** [comment]
This looks good to me, but admittedly I don't know much about mdapi.

**Comment by pypingou** [comment]
I'm going to say, let's merge this but I will not be able to make a release and deploy it before October.

At least this will get this PR out of the queue :)

**Pull request merged**